tests/Log/SimpleLoggerTest.php

```diff
@@ -113,6 +113,10 @@ public function __toString(): string
      */
     public function testPsrLogInterfaceMethods($message, string $expected): void
```

Comment:
```suggestion
    /**
     * @requires PHP < 8.0
     */
    public function testPsrLogInterfaceMethods($message, string $expected): void
```

Reply:
https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests-skipping-tests-examples-databaseclassskippingtest-php